mrp_unbuild_done_message/models/mrp_unbuild.py

```diff
@@ -0,0 +1,25 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+class MrpUnbuild(models.Model):
+    _inherit = "mrp.unbuild"
+
+    def send_notification_email(self):
+
+        if self.state == 'done':
+            mrp_users = self.env.ref('mrp.group_mrp_manager').users
```

Comment:
Checked in AluTest environment, an error is raised if user has no mail, we should exclude them:

![imagen](https://github.com/user-attachments/assets/28a09015-a1f6-455b-928e-b06c71540d14)

Suggested (not tested):
```suggestion
            mrp_users = self.env.ref('mrp.group_mrp_manager').users.filtered(lambda x: x.email)
```